plugins/modules/gitlab_project.py

```diff
@@ -172,6 +172,39 @@
       - This option is only used on creation, not for updates. This is also only used if I(initialize_with_readme=true).
     type: str
     version_added: "4.2.0"
+  builds:
+    description:
+      - C(private) Repository CI/CD allowed only to project members.
```

Comment:
Please write a complete sentence in every paragraph:
```suggestion
      - C(private) means that repository CI/CD allowed only to project members.
```